### PR TITLE
fixed node label bug

### PR DESCRIPTION
**Pull request by mwoodson**
Provided node labels were not being merged into the configmaps.

### CODE REVIEW

File: roles/openshift_node_group/tasks/create_config.yml [hunk 1]
```diff
@@ -42,7 +42,6 @@
       src: "{{ mktempout.stdout }}/node-config.yaml"
       edits: "{{ openshift_node_group_edits | union(openshift_node_labels_edit) }}"
```
Comment (michaelgugino) [inline roles/openshift_node_group/tasks/create_config.yml]: openshift_node_group_edits is empty, and set in vars?  We should just remove this variable then.

File: roles/openshift_node_group/tasks/create_config.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@
   run_once: true
 
 - when:
-  - configout.results.results.0 == {} or (configout.results.results.0 != {} and openshift_node_group_edits|length > 0)
+  - configout.results.results.0 == {} or (configout.results.results.0 != {} and (openshift_node_group_edits|length > 0 or openshift_node_group_labels|length > 0))
```
Comment (michaelgugino) [inline roles/openshift_node_group/tasks/create_config.yml]: This statement will never be true.

Comment (sdodson) [reply roles/openshift_node_group/tasks/create_config.yml]: These get set by main.yml when iterating over openshift_node_groups.

File: roles/openshift_node_group/tasks/create_config.yml (hunk 1 above)
Comment (sdodson) [reply roles/openshift_node_group/tasks/create_config.yml]: I'm fine with that, but we can tackle that later.

**Pull request merged**